server.js

```diff
@@ -70,17 +70,30 @@ app.get("/historical/:country", async function (req, res) {
   res.send(countryData);
 });
 
-app.get("/countries/:country", async function (req, res) {
+app.get("/countries/:search", async (req, res) => {
```

Comment:
query, not search

Reply:
search is for all, ISO, Id and country name, What do you suggest?

Reply:
she's talking about naming, query is better

Reply:
fixed